packages/shared/client/ReplayClient.ts

```diff
@@ -340,6 +340,12 @@ export class ReplayClient implements ReplayClientInterface {
     return countsObject;
   }
 
+  async getFrameSteps(pauseId: PauseId, frameId: FrameId): Promise<PointDescription[]> {
```

Comment:
I wonder if we could name this method something a little more descriptive (for people like me who get confused by it) – maybe even just something like `getFrameStepPoints` or something?

Reply:
Hmm, there's dozens of places in the code where we talk about "FrameSteps", so I'd like to keep the name here for consistency.

Reply:
I guess that's fair but also...isn't it an array of execution points? Do we ever call "points" "steps" except for here?